spectator-web/src/main/java/com/netflix/spectator/controllers/filter/MeasurementFilter.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spectator.controllers.filter;
+
+import com.netflix.spectator.api.Measurement;
+import com.netflix.spectator.api.Meter;
+
+public interface MeasurementFilter {
+  public boolean keep(Meter meter, Measurement measurement);
```

Comment:
I thought filters could potentially be on meters or measurements. I'm filtering on meter names and measurement labels. Although the measurement names are kind of related, I thought one might want to filter on other meter properties such as its type (e.g. ignore distributions).